src/Utils/Lib.Sort.dfy

```diff
@@ -0,0 +1,450 @@
+include "Library.dfy"
```

Comment:
We should eventually move as much as possible of this file and `Library.dfy` into https://github.com/dafny-lang/libraries.